server/grammar/YmlLexer.g4

```diff
@@ -141,7 +142,7 @@ INTEGER: NON_ZERO_DIGIT DIGIT+ | DIGIT+;
  * − double colons are used to define a class' method;
  * − triple colons are used as a pointer towards a class' method or attribute.
  */
-YMLID: ID (COLON ID)* (COLON? COLON? COLON ID)?;
+YMLID: ID (COLON ID)* (COLON? COLON? COLON ID)? SINGLE_QUOTE?;
```

Comment:
What the hell are single quotes doing at the end of IDs? xD Maybe you could update the fat comment above to explain it?

Reply:
I'm not even sure myself what this means. I know it can be used in a `new` instruction.
This instruction creates an object and can be used to set some of its attributes.
In javascript, you would have:

```javascript
{
    attr1: value1, 
    attr2: value2
}
```

In YML, assuming you're building an object of class MyClass,  it would be:
```yseopml
myObj = new(MyClass, attr1, value1, attr2, value2);
```

In case `attr1` and `attr2` are also identifiers of local variables, you can add a single quote to say that you are using the attribute name instead of the local variable.

```yseopml
function myFunction
args { }
local {
    Object attr1
    Object attr2
}
--> action {
// some code
myObj = new(MyClass, attr1', value1, attr2', value2);
// more code
};
```

Reply:
o m g

Reply:
I suppose asking you to edit the grammar so that this `'` would only be authorized in a `new(…)% would be overkill (and maybe even dangerous if you’re not sure whether there exist other valid uses).